crates/release_plz/src/args/update.rs

```diff
@@ -101,8 +102,12 @@ impl RepoCommand for Update {
 }
 
 impl Update {
-    pub fn config(&self) -> anyhow::Result<Config> {
-        super::parse_config(self.config.as_deref())
+    pub fn config(&self, cargo_metadata: &Metadata) -> anyhow::Result<Config> {
+        super::parse_config(
+            self.config.as_deref(),
+            self.optional_project_manifest(),
```

Comment:
project_manifest is already inside `self.config`. You don't need this parameter.

Reply:
`self.config` is a `PathBuf`. Not sure if I understood.

Reply:
oops sorry. Than you can find the project manifest in `cargo_metadata.workspace_root.join("Cargo.toml")`

Reply:
Not sure if we can do this. What happens if the user provided a custom path via `--project-manifest`?

Reply:
Sorry, I missed this comment.
It's already taken into account [here](https://github.com/orhun/release-plz/blob/471de295c9a01684bde6bd647da6f7290a0fc4ee/crates/release_plz/src/main.rs#L29)